pkg/controller/plan/migration.go

```diff
@@ -545,6 +552,31 @@ func (r *Migration) deletePvcPvForOva() (err error) {
 	return
 }
 
+func (r *Migration) deleteValidateVddkJob() (err error) {
```

Comment:
i would prefer to see it in destinationclient or kubevirt, but that's not that important

Reply:
yeah, we're inconsistent about where to place the cleanup pieces (e.g., deleteImporterPods is here and DeletePVCConsumerPod in kubevirt.go)